### PR TITLE
Replaced crypted-preferences...

### PR DESCRIPTION
**Pull request by jason-dgio**
…with the official shared-preferences dependency

### CODE REVIEW

File: lib/src/FauiLocalStorage.dart [hunk 1]
```diff
@@ -34,46 +17,68 @@ class FauiLocalStorage {
...
-      }
-      user = await FbConnector.refreshToken(user: user, apiKey: apiKey);
-      _storeLocally(_LocalKey, jsonEncode(user));
-      FauiAuthState.user = user;
-      print("sso: succeeded silent sign-in");
-      return;
+        FauiUser user = FauiUser.fromJson(jsonDecode(v));
+        if (user == null || user.refreshToken == null) {
+          print("sso: no refresh token found");
+          return;
```
Comment (polina-c) [inline lib/src/FauiLocalStorage.dart]: It seems second print will never happen.

File: lib/src/FauiLocalStorage.dart [hunk 2]
```diff
@@ -34,46 +17,68 @@ class FauiLocalStorage {
...
     print("sso: started silent sign-in");
     try {
-      await _initialize();
-      FauiUtil.throwIfNullOrEmpty(value: apiKey, name: "apiKey");
-      String v = _getLocalValue(_LocalKey);
-      if (v == null || v == "null") {
-        print("sso: no user stored");
-        return;
-      }
+        await _getLocalValue(_LocalKey).then((String v) async {
```
Comment (polina-c) [inline lib/src/FauiLocalStorage.dart]: If you use the keyword 'await', you do not need 'then'.
Next statement will wait for this statement to finish.
There are two styles to manage asynchrony: (1) with 'then' and (2, more modern) with async/await/Future.
In Faui I use the second way. 

See more details here:  https://dart.dev/codelabs/async-await.

Can you remove 'then', please?

**Comment by jason-dgio** [comment]
Sure, that shouldn't have been there, I'll amend

Sent from ProtonMail mobile

-------- Original Message --------

**Comment by polina-c** [comment]
shared_preferences is a plugin.
Flutter web does not support plugins yet.
Holding this PR till plugin support.

**Pull request merged**